openshift-ansible.spec

```diff
@@ -190,6 +201,33 @@ BuildArch:     noarch
 %files lookup-plugins
 %{_datadir}/ansible_plugins/lookup_plugins
 
+# ----------------------------------------------------------------------------------
+# atomic-openshift-utils subpackage
+# ----------------------------------------------------------------------------------
+
+%package -n atomic-openshift-utils
+Summary:       Atomic OpenShift Utilities
+BuildRequires: python-setuptools
+Requires:      ansible
+Requires:      python-click
+Requires:      python-ecdsa
+Requires:      python-jinja2
+Requires:      python-markupsafe
+Requires:      python-paramiko
+Requires:      python-crypto
+Requires:      python-setuptools
+Requires:      PyYAML
+BuildArch:     noarch
```

Comment:
I believe you can remove the deps on ecdsa, jinja2, markupsafe, paramiko, and crypto. I think these are all ansible deps we were bundling before due to the virtualenv use. These should be covered by the ansible dep now.